plugins/cd/_cdguipanel.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, tracks):
 
     def __prepare_transcoder(self):
         formats = transcoder.get_formats()
-        default_format = next(formats.keys())
```

Comment:
I don't really get this one. I thought these were identical in python3.

Reply:
Me too. I am quite sure that it worked with python 3.9 when I tested the plugin last time.